src/huggingface_hub/hf_api.py

```diff
@@ -897,9 +897,9 @@ def model_info(
             else f"{self.endpoint}/api/models/{repo_id}/revision/{revision}"
         )
         headers = {"authorization": f"Bearer {token}"} if token is not None else None
-        status_query_param = "?securityStatus=True" if securityStatus else ""
+        status_query_param = {"securityStatus":True} if securityStatus else None
         path += status_query_param
-        r = requests.get(path, headers=headers, timeout=timeout)
+        r = requests.get(path, headers=headers, timeout=timeout, params=status_query_param)
```

Comment:
Tell me if this works for you @muellerzr

Reply:
@McPatate close! See my suggestion below. Otherwise good to go!